scripts/helpers/set-conf.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+
+function set_yaml_prop () {
+    key="${2}:"
+    full_line="${key} ${3}"
+
+    if grep -q "^#\?\s*${key}" "${1}";
+    then
+        sed -i "s@.*${key}.*@${full_line}@" "${1}"
+    else
+        echo "${full_line}" >> "${1}"
+    fi
+}
```

Comment:
Pretty sure this is duplicated in one of the other files, might be worth just sourcing it, but also note my other comment about staging `yq`

Reply:
the conf_setter is already in python, why not write all of these in python to get the benefit of easier testability?

Reply:
Yeh, that's maybe better than staging someone else's snap... my main reason for suggesting `yq` is to avoid carrying `python` in the snap, but perhaps you could just fetch `yq` directly from the Gtithub release instead? https://github.com/mikefarah/yq/releases

Reply:
I went with yq for the time being, as the properties set / manipulated are currently only dealing with strings and other primitives, once we hit object structures it would make sense to switch to the conf-setter.py that supports json objects as input